docs/source/manifests.md

```diff
@@ -183,8 +185,28 @@ VPCID="vpc-0c4cb9e06c9413222"
 PRIVATESUBNETS=["subnet-0c36d3d5808f67a02","subnet-00fa1e71cddcf57d3"]
 SECURITYGROUPS=["sg-049033188c114a3d2"]
 ```
+(dependency-management)=
+### Dependency Management
 
+SeedFarmer has a shared-responsibilty model for dependency management of modules.  We have put in guardrails within SeedFarmer to prevent accedental deletion of modules that have other modules dependent on it, prevent circular references of modules, etc., but it is up to the end user to be aware of and manage the relationships between modules to assess impact of changes to modules via redeployment.  We recommend to force destroy / deploy / redeploy of modules explicitly via the manifests.
```

Comment:
1. Can you add some light on what type of guardrails seedfarmer has to prevent accidental deletion and how is customer responsible to handle the dependency management force deployment impact?
2. `accedental` spell check

Reply:
Requested changes added / addressed